acceptance/bundle/variables/double_underscore/output.txt

```diff
@@ -0,0 +1,9 @@
+
+>>> $CLI bundle validate
+Name: double_underscore
+Target: default
+Workspace:
+  User: $USERNAME
+  Path: /Workspace/Users/$USERNAME/.bundle/double_underscore/default
```

Comment:
This output doesn't test that it actually works. I suspect this passes without the fix as well.

To assert the variable is correctly interpolated, you can use `jq` on the JSON output to retrieve the field value where it should have been interpolated.

Reply:
Yes, good catch! Missed it when moved the approach from warning to handling the error